cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -65,9 +65,8 @@ def generate_config(self, analysis_obj: Analysis) -> ScoutLoadConfig:
         # Fetch last version from housekeeper
         # This should be safe since analyses are only added if data is analysed
         hk_version_obj: Version = self.housekeeper.last_version(analysis_obj.case.internal_id)
-        LOG.debug("Found housekeeper version %s", hk_version_obj.id)
+        LOG.debug(f"Found housekeeper version {hk_version_obj.id}")
 
-        load_config: ScoutLoadConfig
```

Comment:
ruff would not allow a commit without this removed since it is not used anywhere. If you want it back, it is easy to revert.